modules/apps/collaboration/announcements/announcements-web/src/main/resources/META-INF/resources/announcements/entry_action.jspf

```diff
@@ -20,6 +20,7 @@
 			<portlet:param name="mvcRenderCommandName" value="/announcements/edit_entry" />
 			<portlet:param name="redirect" value="<%= announcementsRequestHelper.getCurrentURL() %>" />
 			<portlet:param name="entryId" value="<%= String.valueOf(entry.getEntryId()) %>" />
+			<portlet:param name="alert" value="<%= String.valueOf(portletName.equals(AnnouncementsPortletKeys.ALERTS)) %>" />
```

Comment:
Shouldn't we add a very similar check here https://github.com/liferay/liferay-portal/blob/master/modules/apps/collaboration/announcements/announcements-web/src/main/resources/META-INF/resources/announcements_admin/entry_action.jsp#L30 ? Otherwise when editing an alert from Site Admin it won't work as expected.